### ISSUE TITLE
reset doesn't change routes

### ISSUE THREAD

**Issue by jpaas** [thebakeryio/react-native-navigation-redux-helpers#4]
It looks like the reset action doesn't actually change the routes, only the index. Isn't the intention to be able to change the routes?

**Comment by lfrodrigues** [comment]
Also think solution for reset in #2 was better. If I have a sack of cards and I want to reset it to one (not popping them) with current version of function this is not possible. 

I can make a PR to have 2 versions of reset, one like you now have and another that does reset to the routes.

**Comment by jpaas** [comment]
Yep that's my use case... I don't care what cards are on the stack, I have a new set of cards.

**Comment by callmephilip** [comment]
Can you guys take fix/route-reset branch for a spin and let me know if it looks good

``` bash
npm install --save git://github.com/bakery/react-native-navigation-redux-helpers.git#fix/route-reset
```

Usage:

``` javascript
reset([{ key: '1'}], navigationKey)
```

**Comment by lfrodrigues** [comment]
Works fine for me, thanks! By the way index is not necessary (I don't pass it)

**Comment by callmephilip** [comment]
re:index - it's needed for compatibility with NavigationStateUtils - https://github.com/facebook/react-native/blob/37e41d2549b3e58f80a5cd075049ac1ddff06033/Libraries/NavigationExperimental/__tests__/NavigationStateUtils-test.js#L141

**Comment by lfrodrigues** [comment]
Sure it makes sense, I just mentioned that in my use case I'm not passing it

**Comment by callmephilip** [comment]
This should be fixed in 0.4.1. Feel free to reopen in case of questions/issues

**Comment by jpaas** [comment]
Thanks @callmephilip. Works perfectly!
